### PR TITLE
Fix for GDI color decoding issues.

### PR DESCRIPTION
**Pull request by akallabeth**
GDI color format is different from bitmap color format for color depths >24.

### CODE REVIEW

**Comment by akallabeth** [comment]
@mfleisz Ok, that should fix your issues.

**Comment by akallabeth** [comment]
@freerdp-bot test and don't nag me...

File: libfreerdp/gdi/gdi.c [hunk 1]
```diff
@@ -363,6 +391,10 @@ UINT32 gdi_get_pixel_format(UINT32 bitsPerPixel)
 		case 8:
 			format = PIXEL_FORMAT_RGB8;
 			break;
+
+		default:
+			format = 0;
```
Comment (mfleisz) [inline libfreerdp/gdi/gdi.c]: @akallabeth Maybe this should Log something (warning/error)?

**Comment by mfleisz** [comment]
@akallabeth Thanks - this fixes coloring issues with nocodec and some legacy RDP drawing functions!

**Comment by mfleisz** [comment]
@freerdp-bot test

**Comment by akallabeth** [comment]
@freerdp-bot test

**Pull request merged**